bookkeeper-server/pom.xml

```diff
@@ -30,16 +30,11 @@
   </properties>
   <dependencies>
     <dependency>
-      <groupId>org.apache.bookkeeper.stats</groupId>
-      <artifactId>bookkeeper-stats-api</artifactId>
+      <groupId>org.apache.bookkeeper</groupId>
+      <artifactId>bookkeeper-common</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
     <dependency>
-      <groupId>com.google.guava</groupId>
```

Comment:
We are shading and relocating guava, I have not checked the final output of the build, is the final result the same as in 4.5, that is that we do not have vanilla "guava" in the classpath ?

Reply:
this is a good catch. shading is a big pain. I would defer this to a later task, because we might need to shade the classes that use guava.

Reply:
Ok  maybe we wil create a new issue. I will take care of think. I am particularly cautious with guava because sometimes at Guava they do not care about compatibility with previous releases